test/unit/response-body.spec.ts

```diff
@@ -1,676 +1,714 @@
-import {customMatchers, CustomMatchers} from './support/custom-jasmine-matchers';
-import {interactionBuilder, pactBuilder} from './support/pact-builder';
-import {swaggerMockValidatorLoader} from './support/swagger-mock-validator-loader';
-import {swagger2Builder} from './support/swagger2-builder';
-import {definitionsBuilder, DefinitionsBuilder} from './support/swagger2-builder/definitions-builder';
-import {operationBuilder} from './support/swagger2-builder/operation-builder';
-import {pathBuilder} from './support/swagger2-builder/path-builder';
-import {responseBuilder} from './support/swagger2-builder/response-builder';
-import {schemaBuilder, SchemaBuilder} from './support/swagger2-builder/schema-builder';
+import { customMatchers, CustomMatchers } from './support/custom-jasmine-matchers';
+import { interactionBuilder, pactBuilder } from './support/pact-builder';
+import { swaggerMockValidatorLoader } from './support/swagger-mock-validator-loader';
+import { swagger2Builder } from './support/swagger2-builder';
+import { definitionsBuilder, DefinitionsBuilder } from './support/swagger2-builder/definitions-builder';
+import { operationBuilder } from './support/swagger2-builder/operation-builder';
+import { pathBuilder } from './support/swagger2-builder/path-builder';
+import { responseBuilder } from './support/swagger2-builder/response-builder';
+import { schemaBuilder, SchemaBuilder } from './support/swagger2-builder/schema-builder';
 
 declare function expect<T>(actual: T): CustomMatchers<T>;
 
-describe('response body', () => {
-    const expectedFailedValidationError = 'Mock file "pact.json" is not compatible with spec file "spec.json"';
+[200, 401, 404].forEach((statusCode) => {
```

Comment:
The real change here is to add multiple `statusCode`, and to show that the response bodies from each of these are checked